test/python/picongpu/quick/picmi/simulation.py

```diff
@@ -285,30 +285,20 @@ def test_operations_simple_density_translated(self):
             # check layout
             if "separate1" in species_names or "colocated1" in species_names:
                 # used "layout"
-                self.assertEqual(3, op.ppc)
+                self.assertEqual(3, op.layout.ppc)
```

Comment:
here ppc is 3, as [above](https://github.com/ComputationalRadiationPhysics/picongpu/pull/5578/changes#diff-7279559bfa16153e85397a461a178c41f935a572cc5734438a24c70a8134b633R154). Is this correlated?